generate_cert.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+cd ~/
```

Comment:
Instead of making it in home directory - can they be made in `./etc/example` - so that users home directory does not get changed

Reply:
> Instead of making it in home directory - can they be made in `./etc/example` - so that users home directory does not get changed

Done